src/commands/new-proj/create-quarkus.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Constructs creation command for Quarkus projects
+ * @param projName desired project name
+ * 
+ * @param kotlin true/false if using Kotlin
+ * 
+ * @param gradle true/false if using Gradle
+ * 
+ * @returns a command that is used to create the Quarkus project
+ */
+export const CreateQuarkus = async (projName : string, kotlin : boolean, gradle : boolean) : Promise<string> => {
+  const orgName = "fi.metatavu";
+  let kotlinS : string = null;
```

Comment:
more readable, do same for kotlinS
```suggestion
  let kotlinString : string = null;
```